ui/src/authentication/state/actions.js

```diff
@@ -33,8 +33,9 @@ export const fetchUserProfile = () => {
   };
 };
 
-export const setAnonymous = () => {
+export const logout = () => {
   return async dispatch => {
+    await authenticationService.logout();
     sessionManager.setAnonymous();
```

Comment:
Should we make this also `logout`?

Reply:
@topless Yes makes sense! Do you want, just for fast merging of this one, to include it in one of the upcoming PRs?

Reply:
@topless I was rethinking a bit this renaming. The `SessionManger` is used as an internal storage reflecting the redux state about user data and if the user is anonymous. In that context wouldn't be clearer to keep the names as it is (`setUser` -> when you have user data, `setAnonymous` -> when user you don't have user data) ?